dht_options.go

```diff
@@ -291,6 +291,16 @@ func RoutingTablePeerDiversityFilter(pg peerdiversity.PeerIPGroupFilter) Option
 	}
 }
 
+// DisableEmptyFindResponses prevents the server from returning responses that have no providers.
+// WARNING: do not change this unless you're using a forked DHT (i.e., a private
```

Comment:
Why do you think this will be a problem on the network? We are avoiding sending a useless message back, and then (I suppose) we are closing the connection. This might even improve the network, right?

Reply:
The responses also include a list of closer peers, even if the provider list is empty.